src/deepsparse/server/system_logging.py

```diff
@@ -42,3 +83,25 @@ def log_request_details(pipeline: Pipeline, **kwargs: Any):
     number of successful inferences times the respective batch size)
     """
     pass
+
+
+def _logging_enabled(server_logger: BaseLogger, group_name: str) -> bool:
```

Comment:
@bfineran what do you think about this private function for enabling and disabling system metric group logging? Maybe we could move this helper function to "higher up" to the server level?

Reply:
In general, we should avoid functions that take the generic logger expecting server structure. Greatly prefer to read the config originally and keep that info in memory so the config is single source of truth